test/Sema/immutability.swift

```diff
@@ -61,6 +61,12 @@ class FooClass {
   mutating            // expected-error {{'mutating' isn't valid on methods in classes or class-bound protocols}} {{3-12=}}
   func baz() {}
 
+  nonmutating         // expected-error {{'nonmutating' isn't valid on methods in classes or class-bound protocols}} {{3-15=}}
```

Comment:
To clarify, before this change, 'nonmutating' on a class func was imprecisely diagnosed as "'mutating' isn't valid on methods in classes or class-bound protocols". After this change, the error is precise: "'nonmutating' isn't valid on methods in classes or class-bound protocols". In other words, no source breaking change.